config/exports/exports.yml

```diff
@@ -147,7 +147,7 @@ scrna_core_cell_extraction_sample_arraying_tube_layout:
     - wells.upstream_tubes
 hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare:
   csv: hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare
-  workflow: scRNA Core Cell Extraction Banking Pooling
+  workflow: scRNA Core Cell Extraction Transfer to Bank Seq and Spare Tubes
```

Comment:
The following were suggested in 007c-lims channel:
For the 'LRC Blood Bank' to 'LRC PBMC Bank' file --> scRNA Core PBMC isolation
For the 'LRC PBMC Bank' to 'LRC Bank Seq' and 'LRC Bank Spare' file --> scRNA Core PBMC Bank

Should we change the workflow to "scRNA Core PBMC Bank" ?